include/libdnf/rpm/sack.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef LIBDNF_RPM_SACK_HPP
+#define LIBDNF_RPM_SACK_HPP
+
+
+#include <solv/pool.h>
```

Comment:
The import is handled by https://github.com/rpm-software-management/libdnf/pull/939/files#diff-b3815f139b0e9dd0f6b875dd024c12f9